### PR TITLE
Google places

### PR DESCRIPTION
**Pull request by benrgross**
organizing map page. need to see some styling from Banu to match other elements. Still need to center items on page. but should be good to work with and help organize the divs

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+50<!DOCTYPE html>
```
Comment (angelinama) [inline index.html:1]: 50? Typo?

**Review by angelinama — CHANGES_REQUESTED** [review]
There is a "50" before html tag, please delete that and I will approve Pull request

File: index.html (hunk 1 above)
Comment (benrgross) [reply index.html]: ok yea i deleted it

Comment (benrgross) [reply index.html]: dont merge yet, i need to resolve somethigng

Comment (benrgross) [reply index.html]: can merge now

Comment (benrgross) [reply index.html]: angelina please merge asap

**Review by angelinama — APPROVED** [review]

**Pull request merged**